HW7/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "hw7",
+  "version": "1.0.0",
+  "description": "ToDo list",
+  "main": "index.js",
+  "scripts": {
+    "server": "webpack-dev-server",
```

Comment:
"webpack-dev-server" is since 5.0 (or so) no longer supported. You have to change it to "webpack serve",